nbpages/converter.py

```diff
@@ -320,6 +321,48 @@ def clean_keyword(kw):
     return kw.strip().title().replace('.', '').replace('/', '').replace(' ', '')
 
 
+def get_changed():
+    '''
+    Returns the names of new/changed notebooks on current commit.
+    Returns .* if any global or .circleci/*  files were changed.
+    Returns None if no files were changed on current commit.
+    '''
+
+    # List of any files that when changed should convert all notebooks
+    global_files = ['convert.py', 'environment.yml', 'index.tpl',
```

Comment:
I think it's a bit dangerous to use specific named files for global state - anyone updating the machinery that uses these files has to know to look here instead.  What about this approach instead:
* If a file is an `.ipynb` file, it's not "global"
* If a file is an a directory that contains an `.ipynb` file, it's not "global"
* If a file is `.rst` or `.md`, it's not "global"
* all other files are "global" and trigger a rebuild of everything

Reply:
Sounds good. Not sure how much time I'll have in the next couple weeks so if you wanted to tackle this in the meantime, otherwise I have some ideas how to do this that I started.